main.go

```diff
@@ -31,6 +34,81 @@ var (
 	outputFileFlag = flag.String("o", "deps.nix", "output nix file")
 )
 
+// FindRealPath queries url to try to locate real vcs path
```

Comment:
It would be nice if you could elaborate a little bit more about the format of the meta tag and how it's being parsed, so the reader does not have too look everything up himself.

Reply:
updated, might be a little too heavy?